src/models/person.js

```diff
@@ -0,0 +1,30 @@
+export default (sequelize, DataTypes) => {
+  const Person = sequelize.define('person', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4
+    },
+    firstName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    lastName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    dni: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    }
+  })
+
+  Person.associate = (models) => {
```

Comment:
Missing association with `user` and physical address with `person`. For the last one, I don't know how to indicate to Sequelize (even, if it's necessary) that an association may not be available (the address may not be associated to a person, but the person does have one or more addresses).